templates/JLink.adapter.json

```diff
@@ -73,16 +90,13 @@
             "name": "<%= pname %> JLink (launch)",
             "type": "gdbtarget",
             "request": "launch",
-            "cwd": "${workspaceFolder}/<%= solution_folder %>",
-            "program": "<%= symbol_files.filter(s => s.pname === pname)[0]?.file || '' %>",
+            "cwd": "<%= data.cwd %>",
+            "program": "<%= data.programs[pname]?.[0].file ?? '' %>",
             "gdb": "arm-none-eabi-gdb",
             "auxiliaryGdb": true,
             "preLaunchTask": "CMSIS Load",
             "initCommands": [
-                "<% const files = symbol_files.filter(s => s.pname === pname).slice(1).map(s => s.file) %>",
-                "<% for (const file of files) { %>",
-                "symbol-file <%= file %>\n",
-                "<% } %>",
+                "<%= data.programs[pname]?.slice(1)?.map(p => p.file ? `symbol-file ${p.file}\\n` : '')?.join('') %>",
```

Comment:
There must be quotes or a sufficient number of backslashes missing somewhere here:
<img width="809" height="65" alt="image" src="https://github.com/user-attachments/assets/773f8eb9-83a3-4ffe-936e-26fce7c8db13" />

Reply:
Looks like Windows Path issue... :(